### DISCUSSION TITLE
Is it possible to use Server-Side Includes for OIDCHTMLErrorTemplate?

### DISCUSSION THREAD

**Discussion by studersi** [OpenIDC/mod_auth_openidc#882] [Q&A] [answered]
As far as I can tell the OIDC error page template is exlcusively templated by OIDCHTMLErrorTemplate. Is there any way to insert more data into the error page like a request ID or IP address?

Generally, this can be achieved with Server-Side Includes but this does not appear to work here.

**Comment by zandbelt** [comment, marked as answer]
that is indeed not supported: you may try and have the error template POST the error and description to another page (.shtml) that does parse SSI (though you may lose the request id)

**Reply by studersi** [reply]
Thank you for the clarification, this answers my question.

Is there any chance, this will be possible in the future?
